lib/travis/logs/database.rb

```diff
@@ -133,6 +133,12 @@ def create_log_part(params)
         db[:log_parts].insert(params.merge(created_at: Time.now.utc))
       end
 
+      def create_log_parts(entries)
+        db[:log_parts].multi_insert(
+          entries.map { |e| e.merge(created_at: Time.now.utc) }
```

Comment:
Should `Time.now` be pulled out into a variable to make sure all log parts have the same timestamp / make fewer system calls?